scripts/cli.py

```
@@ -0,0 +1,340 @@
+    
+    # Check if the string matches the expected pattern
+    pattern = r'^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}$'
+    if not re.match(pattern, date_string):
+        raise click.BadParameter(
+            f"Invalid {field_name} format: '{date_string}'. "
+            f"Expected format: YYYY-MM-DD HH:MM:SS (e.g., '2024-01-15 14:30:00')"
+        )
+    
+    try:
+        return datetime.strptime(date_string, expected_format)
+    except ValueError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)